_episodes/01-short-introduction-to-Python.md

```diff
@@ -450,7 +450,7 @@ for key in rev.keys():
 > ## Changing dictionaries
 >
 > 1. First, print the value of the `rev` dictionary to the screen.
-> 2. Reassign the second value (in the *key value pair*) so that it no longer
+> 2. Reassign the value that corresponds to the key "2" so that it no longer
```

Comment:
Sorry, I forgot to submit my review before. @NPalopoli, we'll need you to click the "commit suggestion" button in your interface to accept the change (if it looks good to you), on either @maxim-belkin or my suggestion. This is because the change is unmerged, and still on your branch, so we don't have access to  your branch to commit changes. Once that's done, I think we can merge this in!

Reply:
Nice! Thank you for the changes and I really appreciate the explanations on the GitHub interface.